languageserver/src/context-providers/secrets.ts

```diff
@@ -28,6 +28,8 @@ export async function getSecrets(
   }
 
   const eventsConfig = workflowContext?.template?.events;
+  const dynamicEnv = workflowContext?.template?.env;
```

Comment:
Only comment is about the warning that dynamicEnv is not used

Reply:
Addressed! 👍